cli50/__main__.py

```diff
@@ -54,6 +54,19 @@ def main():
     except subprocess.TimeoutExpired:
         sys.exit("Docker not responding.")
 
+    # Check for newer version
+    try:
+        latest = max(requests.get("https://pypi.org/pypi/cli50/json").json()["releases"])
```

Comment:
Not sure `max` will work here for semantic versioning? E.g.:

```
>>> max('1.9.0', '1.10.0')
'1.9.0'
```

But [`parse_version`](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#parsing-utilities) might be helpful?

> Parsed a project’s version string as defined by PEP 440. The returned value will be an object that represents the version. These objects may be compared to each other and sorted.

Reply:
https://github.com/cs50/cli50/pull/79/commits/8035e1a34f1e1bdd833875f8fb1f8a59d4a0c735